src/browser-lib/analytics/useAnalyticsService.ts

```diff
@@ -37,18 +36,18 @@ const useAnalyticsService = <T>({
         setLoaded(true);
       }
     };
-    if (consentState === "enabled" && !hasAttemptedInit) {
+    if (consentState === "granted" && !hasAttemptedInit) {
       attemptInit();
     }
   }, [consentState, hasAttemptedInit, config, service, setPosthogDistinctId]);
 
   useEffect(() => {
     // do not track
     if (loaded) {
-      if (consentState === "enabled") {
+      if (consentState === "granted") {
         service.optIn();
       }
-      if (consentState === "disabled") {
+      if (consentState === "granted") {
```

Comment:
Shouldn't this be `consentState === "denied"`?

Reply:
@simonrose121 

![picardfacepalm2](https://github.com/oaknational/Oak-Web-Application/assets/122096/44905480-5451-4eb3-9e00-123457795643)

you're absolutely right. I got a little trigger happy with the copy-paste it seems.

Thanks for catching and i've added a test to exercise that block.

Reply:
All good!